Python/di_sensors/easy_light_color_sensor.py

```diff
@@ -0,0 +1,179 @@
+# https://www.dexterindustries.com
+#
+# Copyright (c) 2018 Dexter Industries
+# Released under the MIT license (http://choosealicense.com/licenses/mit/).
+# For more information see https://github.com/DexterInd/DI_Sensors/blob/master/LICENSE.md
+#
+
+# EASIER WRAPPERS FOR:
+# IMU SENSOR,
+# LIGHT AND COLOR SENSOR
+# TEMPERATURE, HUMIDITY and PRESSURE SENSOR
+
+# MUTEX SUPPORT WHEN NEEDED
+
+
+from di_sensors import light_color_sensor
+from di_sensors import VL53L0X
+import I2C_mutex
+from time import sleep
+from math import sqrt
+
+'''
+MUTEX HANDLING
+'''
+from di_sensors.easy_mutex import *
```

Comment:
Everywhere we have the statement `from di_sensors.easy_mutex import *` we'd better replace the `*` with the functions we're interested in - in this case it's `ifMutexAcquire` and `ifMutexRelease`. 

I'm saying this because it's best to lower the chances of getting into unreasonable conflicts in the future(that are really hard to understand). In `easy_mutex.py` there is:
1. `I2C_Mutex` class that we don't want to inherit -> this class is already imported in this file, so it's redundant to do it twice in a row.
2. `mutex` variable which should not get imported at all -> imagine doing some mutex work in `easy_whatever_sensor.py` file in the future and forgetting about this imported variable -> things could go haywire.
3. `overall_mutex` variable which could pose a smaller chance of creating issues in the future and yet, it's something we don't want to import.
4. Whatever variable that can get into conflict with whatever is/(will be) defined in our `easy_whatever_sensor.py`.

Reply:
reason I went with `import *` is that all of those lines were already in  each of the files.

Reply:
But now, they are separate, which means they could "evolve" separately. 

Should someone else come and edit this library, they could make the simple mistake of ignoring one of the files and thus create conflicts. 
It's always best to be safe than sorry.